hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseCryptoHandler.java

```diff
@@ -56,4 +57,9 @@ protected long getStakedId(
     public static AccountID asAccount(final long num) {
         return AccountID.newBuilder().accountNum(num).build();
     }
+
+    @NonNull
+    public static TokenID asToken(final long num) {
+        return TokenID.newBuilder().tokenNum(num).build();
+    }
```

Comment:
I think this method already exists in `BaseTokenHandler`

Reply:
removed.